src/Utopia/Messaging/Adapters/SMS/Clickatell.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+// Reference Material
+// https://docs.clickatell.com/channels/sms-channels/sms-api-reference/#tag/SMS-API/operation/sendMessageREST_1
+class Clickatell extends SMSAdapter
+{
+    /**
+     * @param  string  $apiKey Clickatell API Key
+     */
+    public function __construct(
+  private string $apiKey,
+ ) {
+    }
+
+    public function getName(): string
+    {
+        return 'Clickatell';
+    }
+
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1000;
```

Comment:
I notice the maximum is set to 1000 for all the new adapters, please double check the max values are correct for all services

Reply:
I was able to verify as follow:
**Plivo** - up to 1000
https://www.plivo.com/docs/sms/api/message#bulk-messaging

**Clickatell** - up to 500
https://6194477.fs1.hubspotusercontent-na1.net/hubfs/6194477/SMS%20REST%20API%20v2.0-1.pdf

**Sinch** - up to 1000
https://developers.sinch.com/docs/sms/api-reference/sms/tag/Batches/

About Seven & Infobin there isn't any information about there _limits_ so I left it on 1,000